src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/AncmIISExpressV2.wixproj

```diff
@@ -51,6 +53,15 @@
     </ProjectReference>
   </ItemGroup>
 
+  <Target Name="CopyBuildOutputToArtifactDirectory"
```

Comment:
Not fond of this approach where I just copy the output like this, but didn't think of a better way.

Reply:
Suggest instead including the relevant files in the build artifacts and doing everything necessary in our build pipeline. Work w/ @aspnet/build to make the pipeline changes once the artifacts are available.

That is, change YAML rather than MSBuild.

Reply:
I disagree. If I change the YAML, I would need to point to a folder path that brings in src\servers\IIS... and do that for multiple folders. It doesn't seem like a great solution.

I also have concerns that the pdbs/dlls that we would get from the end of build won't match the build from the installers. I already tried to get pdbs from a different component than the installers and they didn't match, meaning during a complete build of aspnetcore, native dlls get rebuilt multiple times and those rebuilds actually change the dll. This is a separate concern that I would like to resolve later, but for the mean time, the safest option is to make so immediately after building the installers, we grab the dlls/pdbs and put them in the artifacts to guarantee they match.

This is a very time sensitive change. If this is a large enough concern to block merging this PR, then I can figure out a better way to fix it. But I don't see much benefit.